ferrocene/doc/public-docs-warning/header.html

```diff
@@ -0,0 +1,9 @@
+<!-- SPDX-License-Identifier: MIT OR Apache-2.0 -->
+<!-- SPDX-FileCopyrightText: The Ferrocene Developers -->
+
+<div class="ferrocene-public-docs-warning hidden">
+    <div class="ferrocene-public-docs-warning-wrapper">
+        <span>This is a preview from our development branch. It might be incomplete or inaccurate.</span>
```

Comment:
My initial thought was that this should link to the "next" release notes to show the major changes in this branch compared to last stable, but I don't think I'm particularly married to the idea.